functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -323,3 +326,10 @@ def get_user_req_body(req):
         pass
 
     return user_req_body
+
+def canInvalidate(cache, user_id):
```

Comment:
you can just return this => 

`    return cache is not None and int(user_user_id) == int(user_id)
`

Reply:
Done. Thanks for that!